frontend/src/components/signup.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import ReactDOM from 'react-dom/client';
+import { useState } from "react"; 
+import axios from 'axios';
+import { fchmod } from 'fs';
+import '../components/signup.css'
+
+export const SignUp = () => {
+
+    const [firstName, setFirstName] = useState<string>('');
+    const [lastName, setLastName] = useState<string>('');
+    const [email, setEmail] = useState<string>('');
+    const [password, setPassword] = useState<string>('');
+    const [confirmPassword, setConfirmPassword] = useState<string>('');
+    const [warning, setWarning] = useState<string>(''); 
+
+    const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+
+        e.preventDefault(); 
+
+            if(password === confirmPassword) {
+                setWarning('');
+                await axios.post('http://localhost:8000/signup', {
+                    firstName,
+                    lastName,
+                    email,
+                    password, 
+                })
+        
+                .then((res) => {
+                    console.log(res.data)
```

Comment:
No change needed for this PR since i think we're going to add more features -- but what should we do after a signup is successful?

Reply:
perhaps send a 200

Reply:
well this would be the frontend, so we're not dealing with status codes! is there a sensible _page_ or something that the user should see?